code/modules/keybindings/bindings_client.dm

```diff
@@ -5,12 +5,13 @@
 	set hidden = TRUE
 
 	keys_held[_key] = world.time
-	var/movement = SSinput.movement_keys[_key]
+	var/focus_chat = prefs.focus_chat
```

Comment:
Actually, you should profile this, if creating a new variable is faster than accessing it twice.

Reply:
![image](https://user-images.githubusercontent.com/1134201/63640462-8d550a00-c698-11e9-83c8-f67f753e6d13.png)

Original vs revised.